CHANGES.rst

```diff
@@ -92,6 +92,9 @@ other
 
 - add `opencv-python` to hard dependencies for usage of snowball detection in the jump step in `stcal` [#7499]
 
+- Remove use of deprecated ``pytest-openfiles`` ``pytest`` plugin. This has been replaced by
```

Comment:
This new entry needs to go into the unreleased (1.10.1) section at the top of the log (version 1.10.0 has already been released)